config/application.rb

```diff
@@ -12,7 +12,9 @@
 
 module LocalSupport
   class Application < Rails::Application
-
+    config.action_view.field_error_proc = Proc.new do |html_tag, instance|
```

Comment:
html in a config file makes me a little uncomfortable - could this be moved to a helper?